components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx

```diff
@@ -172,6 +181,8 @@ export default class CustomTimeRangeSelector extends React.Component {
                         />
                     </div>
                 </div>
+                {this.state.inValidDateRange ? <div style={{color: '#dc3545', paddingTop: 10}}>
+                    Invalid date range, Please select a valid date range. </div> : ''}
```

Comment:
Shall we format this for better reading of ternary operator.

Reply:
This code has been completely rewritten. https://github.com/wso2/analytics-solutions/pull/270